src/HyperscriptLiteral.jl

```diff
@@ -52,25 +76,25 @@ parseelems(io::IO; interp::Bool=false) = parseelems(io -> true, io; interp=inter
 function parseelems(predicate, io::IO; interp::Bool=false)
     elems = Any[]
     parseelems!(predicate, io, elems; interp=interp)
-
     return elems
 end
 function parseelems!(predicate, io::IO, elems::AbstractVector; interp::Bool=false)
     while !eof(io) && predicate(io)
-        push!(elems, peek(io, Char) == '<' ? parsetag(io; interp=interp) : parsechars(io; interp=interp))
+        push!(elems, parseelem(io; interp=interp))
     end
 end
 
 """
-    parsechars(io::IO)
+    parseelem(io::IO)
 
-Parse a text element.
+Parse a single HTML element.
 """
-function parsechars(io::IO; interp::Bool=false)
-    chars = readuntil(io, '<')
-    !eof(io) && skip(io, -1)
-
-    return chars
+function parseelem(io::IO; interp::Bool=false)
+    # TODO: revisit this implementation after.
+    c = peek(io, Char)
```

Comment:
Changing `peek(io, Char)` to `Char(peek(io))` might work with Julia 1.4 (two-argument `peek` appeared in 1.5).